tests/test_pretty.py

```diff
@@ -67,6 +67,7 @@ class ExampleDataclass:
     bar: str
     ignore: int = field(repr=False)
     baz: List[str] = field(default_factory=list)
+    _ignore: int = field(init=False, repr=False)
```

Comment:
What is this for?

Reply:
This would cause the test to fail if the dataclass pretty repr didn't handle the last field having repr=false properly. Which was the issue, it would have a trailing comma and whitespace.

Reply:
This caused the test to fail without the changes proposed for pretty.py